dashboard/src/components/Hint/Hint.scss

```diff
@@ -0,0 +1,14 @@
+.extraClass {
```

Comment:
Why do you need to apply `!important` to all the CSS classes? Please, avoid the usage of `!important` when it's not required because it breaks the CSS cascade feature.

Reply:
I think so. Probably they're adding the `!important` _just in case_